src/System.Management.Automation/FormatAndOutput/common/FormattingObjects.cs

```diff
@@ -23,6 +23,7 @@
 //
 
 using System.Collections.Generic;
+using System.Management.Automation;
```

Comment:
Is it necessary to add this?

Reply:
Yes, since I moved the WriteStreamType enum